### PR TITLE
fix: missing static generation store in revalidation due to pages dir

### PR DESCRIPTION
**Pull request by ShubhamPalriwala**
## What does this PR do?
We had 3.2k runtime errors in last 14 days in the `/api/v1/client/environments/<env-id>/people` endpoint. This PR fixes it. It was caused because the `revalidateTag` does not exactly work the same way for pages directory as it does for app directory. This PR migrates the given endpoint to app dir.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
Hit the endpont, get no errrors 💪🏼 

### Required

- [ ] Filled out the "How to test" section in this PR
- [ ] Read [How we Code at Formbricks](<[https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md](https://formbricks.com/docs/contributing/how-we-code)>)
- [ ] Self-reviewed my own code
- [ ] Commented on my code in hard-to-understand bits
- [ ] Ran `pnpm build`
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues
- [ ] First PR at Formbricks? [Please sign the CLA!](https://formbricks.com/clmyhzfrymr4ko00hycsg1tvx) Without it we wont be able to merge it 🙏

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/web/app/api/v1/client/environments/[environmentId]/people/route.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+export async function POST(
+  _: Request,
+  { params }: { params: { environmentId: string } }
+): Promise<NextResponse> {
+  const { environmentId } = params;
+
+  if (!environmentId) {
+    return responses.badRequestResponse("Missing environmentId");
```
Comment (mattinannt) [inline apps/web/app/api/v1/client/environments/[environmentId]/people/route.ts:18]: Please make sure to set the flag to enable CORS

File: apps/web/app/api/v1/client/environments/[environmentId]/people/route.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    return responses.successResponse(
+      {
+        person,
+        session,
+        settings,
+      },
+      true
+    );
+  } catch (error) {
+    return responses.internalServerErrorResponse(error.message);
```
Comment (mattinannt) [inline apps/web/app/api/v1/client/environments/[environmentId]/people/route.ts:35]: Please make sure to set the flag to enable CORS

**Review by mattinannt — CHANGES_REQUESTED** [review]

**Comment by ShubhamPalriwala** [comment]
Done, please take a look now @mattinannt

**Review by mattinannt — APPROVED** [review]
🙏

**Pull request merged**